cram_moveit/src/moveit.lisp

```diff
@@ -588,11 +668,54 @@ as only the final configuration IK is generated."
               planning-group pose-stamped
               :allowed-collision-objects allowed-collision-objects
               :plan-only t
+              :path-constraints-msg path-constraints-msg
+              :start-state start-robot-state
               :touch-links touch-links
               :ignore-collisions ignore-collisions
               :max-tilt max-tilt)))))
 
+(defun plan-link-movements (link-name planning-group poses-stamped
```

Comment:
Does this function reflect the exact same behavior as the one deleted further up? Because parts of `pr2_manipulation_process_module` depend on it, I think.